utopian-io/v2.utopian.io
# Feature/comments

### Verify that
Git checklist

Front end checklist

API checklist

Tests and comments checklist

### My PR doesn't concern a Jira task
Please write a description of what this PR is about before providing the test plan.

---

### New feature: article comments

This PR implements comment creation and displaying on the article page

Comment voting and comment editing/deleting will be handled by a separate PR

### **Test plan**

* Go to http://localhost:8080/en/articles/adriel/another-test-1547586974631
* A list of comments should appear
* Add another one
* The new comment should be displayed in the top
* Reload the page
* The comments list should be correct
